dev/com.ibm.ws.webcontainer.security/src/com/ibm/ws/webcontainer/security/ReferrerURLCookieHandler.java

```diff
@@ -113,8 +110,22 @@ public Cookie createCookie(String cookieName, @Sensitive String value, boolean e
 
             if (sameSite.equals("None")) {
                 c.setSecure(true);
+				Boolean partitioned = webAppSecConfig.getPartitionedCookie();
+                if (partitioned!=null) {
+				    //web container wants the value as a n/v pair
+                    requestState.setCookieAttributes(cookieName, "Partitioned="+partitioned.toString());
+                }
             }
-        }
+		} else if (sameSite == null) {
```

Comment:
sameSite == null || sameSite.equals("Disabled")

Reply:
Doesn't that make it just a regular **else**?